src/components/TransactionsTable.jsx

```diff
@@ -21,6 +23,7 @@ const transactionUSDValue = ({ direction, daiAmount }) => (
 
 const TransactionsTable = () => {
   const transactions = (myAccount.awpTransactions || []).sort((a, b) => b.timestamp - a.timestamp);
+  const { chainId } = useWeb3React();
```

Comment:
Please standardize around the `eth` store instead. `eth.networkId`. We don't always fully initialize web3React.